File: BennuLib.Sharp/PivotPoint.cs
using System;

namespace BennuLib
{
	[Serializable()]
	public struct PivotPoint
	{
		public readonly int Id;
		public readonly int X;

		public readonly int Y;
		public PivotPoint(int id, int x, int y)
		{
			this.Id = id;
			this.X = x;
			this.Y = y;
		}
	}
}

File: BennuLib/PivotPoint.vb
<Serializable>
Public Structure PivotPoint
    Public ReadOnly Id As Integer
    Public ReadOnly X As Integer
    Public ReadOnly Y As Integer

    Public Sub New(id As Integer, x As Integer, y As Integer)
        Me.Id = id
        Me.X = x
        Me.Y = y
    End Sub
End Structure
